### PR TITLE
Support paused eus upgrade using --to-latest option

### PR DESCRIPTION
**Pull request by shellyyang1989**

### CODE REVIEW

**Comment by shellyyang1989** [comment]
cc @jiajliu @jianlinliu @liangxia @jhou1 for review. Thanks!

File: ci-operator/step-registry/cucushift/upgrade/paused/intermediate/to-latest/cucushift-upgrade-paused-intermediate-to-latest-commands.sh [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            sleep 5m
+            (( wait_upgrade -= 5 ))
+        fi        
+    done
+    if (( wait_upgrade <= 0 )); then
+        echo >&2 "Upgrade timeout, exiting" && return 1
+    fi
+}
+
+function run_command_oc() {
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/upgrade/paused/intermediate/to-latest/cucushift-upgrade-paused-intermediate-to-latest-commands.sh:90]: No place to reference this function?

Comment (shellyyang1989) [reply ci-operator/step-registry/cucushift/upgrade/paused/intermediate/to-latest/cucushift-upgrade-paused-intermediate-to-latest-commands.sh:90]: Thanks! Updated.

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/upgrade/paused/intermediate/to-latest/cucushift-upgrade-paused-intermediate-to-latest-commands.sh:90]: How about rename the function name to `run_command_oc_retries`

Comment (shellyyang1989) [reply ci-operator/step-registry/cucushift/upgrade/paused/intermediate/to-latest/cucushift-upgrade-paused-intermediate-to-latest-commands.sh]: Thanks for the suggestion! Updated.

**Comment by shellyyang1989** [comment]
cc @liangxia @jhou1 for approval. Thanks!

**Pull request merged**